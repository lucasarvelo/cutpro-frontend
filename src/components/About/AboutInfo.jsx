import React from "react";

function AboutInfo() {
  return (
    <div>
      <p>
        Proof of concept for an app to admin product orders for a windows
        production company and generate optimized cutting list for windows
        parts.
      </p>
      <p>Build by Lucas Arvelo</p>
    </div>
  );
}

export default AboutInfo;
