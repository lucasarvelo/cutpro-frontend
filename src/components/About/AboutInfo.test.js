import React from 'react';
import AboutInfo from './AboutInfo';
import renderer from 'react-test-renderer';

test('About information is render', () => {
  const component = renderer.create(<AboutInfo />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
