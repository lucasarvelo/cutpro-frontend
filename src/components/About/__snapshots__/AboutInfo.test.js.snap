// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info is render 1`] = `
<div>
  <p>
    Proof of concept for an app to admin product orders for a windows production company and generate optimized cutting list for windows parts.
  </p>
  <p>
    Build by Lucas Arvelo
  </p>
</div>
`;
